chapters/synchronous.tex

```diff
@@ -1118,7 +1118,7 @@ \section{Discretized Partition}\label{continuous-time-equations-in-clocked-parti
 However, even if relations do not generate events, when-clauses and operators \lstinline!edge! and \lstinline!change! should behave as normal.
 \end{nonnormative}
 
-From the view of the clocked partition, the continuous-time partition is discretized and the discretized continuous-time variables have only a value at a clock tick.
+From the view of other partitions, the continuous-time partition is discretized and the discretized continuous-time variables have only a value at a clock tick.
 Therefore, such a partition is handled in the same way as any other clocked partition.
```

Comment:
This is an old problem, but we should try to resolve it now: To me the second sentence here seems to say that "other clocked partitions are handled the same way as any other clocked partition".  I'm sure the intention is something more meaningful.

Reply:
Tried to clarify that it is the outside view that is identical.